components/EditableTitle.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+import styles from "../layouts/default.layout.module.scss";
+
+export const EditableTitle = (props) => {
+  const { defaultValue, readOnly, onConfirm } = props;
+  const [projectTitle, setProjectTitle] = useState(defaultValue);
+
+  const handleConfirm = (e) => {
+    if (e.code === "Enter") {
+      e.preventDefault();
+      e.target.blur();
+    }
+  };
+
+  useEffect(() => {
+    setProjectTitle(defaultValue);
+  }, [defaultValue]);
+
+  return (
+    <input
```

Comment:
I think this is more of an intuitive alternative to edit the title for people who are used to this type of feature from other websites such as Figma. I also feel like changing it into an input field kind of make it looks less like a title and more like something that needs to be submitted?

Reply:
Okay. Could you mark the process name when a user clicks on it so that it's more visible? That's how Figma has it.

Reply:
It could also be possible to use the TextField component from EDS which will make everyone aware of that it's possible to edit the process title directly.